examples/mcp/src/sse/server.ts

```diff
@@ -14,6 +14,7 @@ mcpServer.tool(
   'check-product-stock',
   'Check if a product is available',
   {
+    // @ts-ignore
```

Comment:
why?

Reply:
need to dig to confirm, i believe it's due to a regression in their latest version!

Reply:
incorrect^ it was due to mismatch in package versions! removed!